tests/Backend/CommonPart1/TableWithConfigurationLoadTest.php

```diff
@@ -570,4 +610,93 @@ public function testLoadIncrementalFromFileToTable(): void
         $events = $this->listEventsFilteredByName($this->client, 'storage.tableImportDone', $tableId, 10);
         $this->assertCount(1, $events);
     }
+
+    public function testLoadFromWorkspaceToTable(): void
+    {
+        $tableName = 'custom-table-3';
+
+        $json = /** @lang JSON */
+            <<<JSON
+{
+  "action": "generate",
+  "backend": "synapse",
+  "operation": "importFull",
+  "source": "table",
+  "columns": [
+    "id",
+    "NAME"
+  ],
+  "primaryKeys": [
+    "id"
+  ],
+  "output": {
+    "queries": [
+      {
+        "sql": "CREATE TABLE {{ id(destSchemaName) }}.{{ id(stageTableName) }} ([id] NVARCHAR(4000), [NAME] NVARCHAR(4000)) WITH (DISTRIBUTION = ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX)",
+        "description": ""
+      },
+      {
+        "sql": "INSERT INTO {{ id(destSchemaName) }}.{{ id(stageTableName) }} ([id], [NAME]) SELECT [id], [NAME] FROM {{ id(sourceSchemaName) }}.{{ id(sourceTableName) }}",
+        "description": ""
+      },
+      {
+        "sql": "CREATE TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} WITH (DISTRIBUTION=ROUND_ROBIN,CLUSTERED COLUMNSTORE INDEX) AS SELECT a.[id],a.[NAME] FROM (SELECT COALESCE([id], '') AS [id],COALESCE([NAME], '') AS [NAME], ROW_NUMBER() OVER (PARTITION BY [id] ORDER BY [id]) AS \"_row_number_\" FROM {{ id(destSchemaName) }}.{{ id(stageTableName) }}) AS a WHERE a.\"_row_number_\" = 1",
+        "description": ""
+      },
+      {
+        "sql": "RENAME OBJECT {{ id(destSchemaName) }}.{{ id(destTableName) }} TO {{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      },
+      {
+        "sql": "RENAME OBJECT {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }} TO {{ id(destTableName) }}",
+        "description": ""
+      },
+      {
+        "sql": "DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      },
+      {
+        "sql": "IF OBJECT_ID (N'{{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }}', N'U') IS NOT NULL DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp') }}",
+        "description": ""
+      },
+      {
+        "sql": "IF OBJECT_ID (N'{{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}', N'U') IS NOT NULL DROP TABLE {{ id(destSchemaName) }}.{{ id(destTableName ~ rand ~ '_tmp_rename') }}",
+        "description": ""
+      }
+    ]
+  }
+}
+JSON;
+
+        [$tableId,] = $this->createTableWithConfiguration($json, $tableName, 'outputMappingFullLoad');
+
+        $this->loadTableFromWorkspace(
```

Comment:
Je pitomé, že z tohohle nevidím, co v té tabulce je a v tom testu není vidět, co se vlastně teda stane, když dám "load from workspace". Ale je to tak i u loadu ze souboru, tak co už :)